docs/user-guide/examples/irso/kustomization.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: ironic-standalone-operator-system
+
+resources:
+- https://github.com/metal3-io/ironic-standalone-operator/releases/latest/download/install.yaml
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+- name: ironic-operator-config
+  literals:
+  - IPA_BASEURI=http://192.168.222.1
+
+patches:
+- target:
+    kind: Deployment
+    name: ironic-standalone-operator-controller-manager
+  patch: |-
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: ironic-standalone-operator
+    spec:
+      template:
+        spec:
+          containers:
+            - name: manager
+              envFrom:
+                - configMapRef:
+                    name: ironic-operator-config
```

Comment:
This patch is no longer needed in IrSO v0.7. Then we have a predictable name for the configmap that we can reuse instead. See https://github.com/metal3-io/ironic-standalone-operator/pull/405